src/client/pages/Varen/Varen.tsx

```diff
@@ -0,0 +1,170 @@
+import { useMemo } from 'react';
+
+import styles from '../Vergunningen/Vergunningen.module.scss';
+import { AppRoutes } from '../../../universal/config/routes';
+import { isError, isLoading } from '../../../universal/helpers/api';
+import { defaultDateFormat } from '../../../universal/helpers/date';
+import { getCustomTitleForVergunningWithLicensePlates } from '../../../universal/helpers/vergunningen';
+import { CaseType } from '../../../universal/types/vergunningen';
+import {
+  ErrorAlert,
+  Linkd,
+  MaintenanceNotifications,
+  PageContent,
+  PageHeading,
+  SectionCollapsible,
+  Table,
+  ThemaIcon,
+  addTitleLinkComponent,
+} from '../../components';
+import { OverviewPage } from '../../components/Page/Page';
+import { ThemaTitles } from '../../config/thema';
+import { useAppStateGetter } from '../../hooks/useAppState';
+import { hasMultiplePermits } from '../VergunningDetail/WVOS';
+import { Vergunning } from '../../../server/services';
+
+export const DISPLAY_PROPS = {
+  identifier: 'Kenmerk',
+  title: 'Soort vaar',
+  dateRequest: 'Aangevraagd',
+};
+
+export const DISPLAY_PROPS_HISTORY = {
+  identifier: 'Kenmerk',
+  title: 'Soort vaar',
+  decision: 'Resultaat',
+};
```

Comment:
Kun je dit in een thema config file zetten zoals bv `HLI-thema-config.ts` en deze gegevens beschikbaar maken via een useVareThemaData hook zoals `useHliThemaData`